src/actions/guides/getting-started/installing.cr

```diff
@@ -162,9 +174,15 @@ class Guides::GettingStarted::Installing < GuideAction
 
     This will let you use `lucky` from the command line.
 
+    macOS (Intel)
     ```plain
     cp bin/lucky /usr/local/bin
     ```
+    
+    macOS (M1)
+    ```plain
+    cp bin/lucky /opt/homebrew/bin
```

Comment:
This section isn't the homebrew installation. This is when you build the lucky binary by hand. I believe it should still be put in to `/usr/local/bin` even on M1, right?

Reply:
Hey @jwoertink On M1 macs, `/usr/local` is owned by root, not the regular user, so this command errors out with permission denied

Reply:
We were having a little discussion on this over in the Lucky discord if you're interested https://luckyframework.org/chat

I think my main hesitation here is that if you're not using homebrew, and you want to install Lucky manually, you wouldn't want to copy to `/opt/homebrew/bin` because then you'd have to manually create that directory and add it to $PATH first. However, I also think that someone not using homebrew would probably be the edge case anyway, so maybe it's fine?... 

I'm also thinking, maybe we just say `sudo cp bin/lucky /usr/local/bin`. I looked at mine on Pop_OS, and it's owned by `root` except for the `lucky ` binary which is owned by me. @robacarp made the suggestion that maybe that step is outside of the scope here? We could  maybe change it to just say "Move the `bin/lucky` binary to a directory in your $PATH so it's accessible anywhere.". Something along those lines...

What do you think @grvm ?

Reply:
Yeah, that makes sense. I'll revert it to the original command with sudo & add a line "or anywhere in your path"